ansible/roles/dashmate/templates/dashmate.json.j2

```diff
@@ -0,0 +1,240 @@
+            },
+            "genesis": {
+              "genesis_time": "2021-07-22T12:57:05.429Z",
+              "chain_id": "dash-{{ ( 'devnet-' + dash_devnet_name if dash_network == 'devnet' else dash_network ) if tenderdash_chain_id is not defined else tenderdash_chain_id }}",
+              "initial_core_chain_locked_height": {{ platform_initial_core_chain_locked_height | int }},
+              "consensus_params": {
+                "block": {
+                  "max_bytes": "22020096",
+                  "max_gas": "-1",
+                  "time_iota_ms": "5000"
+                },
+                "evidence": {
+                  "max_age": "100000",
+                  "max_age_num_blocks": "100000",
+                  "max_age_duration": "172800000000000"
+                },
+                "validator": {
+                  "pub_key_types": [
+                    "bls12381"
+                  ]
+                },
+                "version": {
+                  "app_version": "1"
+                }
+              },
+              "validator_quorum_type": {{ platform_drive_validator_set_llmq_type }}
+            }
+          }
+        },
+        "dpns": {
+          "contract": {
+            "id": "GWRSAVFMjXx8HpQFaNJMqBV7MBgMK4br5UESsB4S31Ec"
+          },
+          "ownerId": "4EfA9Jrvv3nnCFdSf7fad59851iiTRZ6Wcu6YVJ4iSeF",
+          "masterPublicKey": "{{ dpns_hd_master_public_key }}",
+          "secondPublicKey": "{{ dpns_hd_second_public_key }}"
+        },
+        "dashpay": {
+          "contract": {
+            "id": "Bwr4WHCPz5rFVAD87RqTs3izo4zpzwsEdKPWUT1NS1C7"
+          },
+          "masterPublicKey": "{{ dashpay_hd_master_public_key }}",
+          "secondPublicKey": "{{ dashpay_hd_second_public_key }}"
+        },
+        "featureFlags": {
+          "contract": {
+            "id": "HY1keaRK5bcDmujNCQq5pxNyvAiHHpoHQgLN5ppiu4kh"
+          },
+          "ownerId": "H9sjb2bHG8t7gq5SwNdqzMWG7KR6sf3CbziFzthCkDD6",
+          "masterPublicKey": "{{ feature_flags_hd_master_public_key }}",
+          "secondPublicKey": "{{ feature_flags_hd_second_public_key }}"
+        },
+        "sourcePath": null,
+        "masternodeRewardShares": {
+          "contract": {
+            "id": "rUnsWrFu3PKyRMGk2mxmZVBPbQuZx2qtHeFjURoQevX"
+          },
+          "masterPublicKey": "{{ mn_reward_shares_hd_master_public_key }}",
+          "secondPublicKey": "{{ mn_reward_shares_hd_second_public_key }}"
+        },
+        "withdrawals": {
+          "contract": {
+           "id": "4fJLR2GYTPFdomuTVvNy3VRrvWgvkKPzqehEBpNf2nk6"
+          },
+          "masterPublicKey": "{{ withdrawals_hd_master_public_key }}",
+          "secondPublicKey": "{{ withdrawals_hd_second_public_key }}"
+        }
+      },
+      "dashmate": {
+        "helper": {
+          "docker": {
+            "image": "dashpay/dashmate-helper:0.24-dev"
+          }
+        }
+      },
+      "externalIp": "{{ dashd_externalip }}",
+      "network": "devnet",
```

Comment:
I think this should go from network name

Reply:
Isn't this a type of network? like `testnet`, `local`, or `mainnet`

Reply:
Correct!